components/CustomTable/CustomTable.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import React from 'react';
+import { Table } from 'antd';
+
+interface DataType {
```

Comment:
You should give it a more meaningful name. This name is so general.

Reply:
I don't understand, could you explain this issue more clearly? Can you give me an example?

Reply:
I mean the interface name. You should give it a name that best describes your data.